api/fact-api-stubs.ts

```diff
@@ -13,6 +13,10 @@ const moneyServiceData = require('./money-service-data.json');
 const serviceAreas = require('./service-areas.json');
 const port = 8080;
 
+const getServiceArea = (serviceArea: string) => {
+  const serviceAreasData = [...serviceAreas];
```

Comment:
this kind of looks weird, why are you using the spread operator here surely serviceAreasData would be the same as serviceAreas

Reply:
it was incase other calls changed something but it doesnt look like the case so ill remove it